handbook/product/README.md

```diff
@@ -111,11 +111,17 @@ Bugs are always prioritized. (Fleet takes quality and stability [very seriously]
 
 If a bug is unreleased or [critical](https://fleetdm.com/handbook/engineering#critical-bugs), it is addressed in the current sprint. Otherwise, it may be prioritized in the Estimated for the next sprint. If a bug [requires drafting](https://fleetdm.com/handbook/engineering#in-product-drafting-as-needed) to determine the expected functionality, the bug should undergo [expedited drafting](#expedited-drafting). 
 
-If a bug is not addressed within 6 weeks, [it is sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). During the sprint kickoff review ritual, each product group PM should review every bug which is marked in this way with the Head of Product and recommended whether a bug should be kept for the next sprint or de-prioritized. A de-prioritized bugs might include action items such as documentation of the defect. The Head of Product then reviews the bug recommendations with the CEO in the bug de-prioritization review ritual. 
+If a bug is not addressed within six weeks, it is [sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). Each sprint, the Head of Product meets with the group product managers to review these bugs (churned bug review). In this session, bugs are categorized as follows:
+- **Continue**: the bug should be prioritized in the next sprint.
+- **Needs prioritization**: the bug will likely not be worked on within the next six weeks, but it is still a valid bug.
+- **De-prioritized**: the issue will be closed and the necessary subsequent steps will be initiated. This might include updating documentation and informing the community.
 
-If a bug should be kept for the next sprint, the group PM should
-- Remove the :product label.
-- Move the bug to the "Inbox" column on the bugs board and assign it to the appropriate EM.
+After aligning with the group product managers, the Head of Product meets with the CEO and Director of Product Development to discuss and finalize the outcomes for the churned bugs.
+
+Once outcomes have been approved by the CEO, Product Operations will complete the churned bug clean-up ritual. Below are the steps for each category:
```

Comment:
:+1: Merging now, but I think we can phase me out of this in favor of Luke once we've done one of these sessions together with Luke and Mo and Mike and made sure everyone's good.  I also think we need both Luke and Mo at customer voice every week first before we make that change, so we can be confident both of you have context for priorities and impact.

@lukeheath @zhumo